src/commands/Hello.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  CommandInteraction,
+  Client,
+  ApplicationCommandType,
+  InteractionResponse,
+} from 'discord.js';
+
+import { Command } from '../interfaces/Command';
+
+export const Hello: Command = {
+  name: 'hello',
+  description: 'Returns a greeting',
+  type: ApplicationCommandType.ChatInput,
+  run: (
+    client: Client,
+    interaction: CommandInteraction,
+  ): Promise<InteractionResponse<boolean>> => {
+    const username = interaction.user?.username;
+    const content = `Ara ara... Hi, ${username}-kun!`;
+
+    return interaction.reply({
+      ephemeral: true,
+      content,
+    });
+  },
+};
```

Comment:
ditto, also we could turn this into `as const satisfies` and achieve better typing..
```suggestion
import { ApplicationCommandType } from 'discord.js';

import { Command } from '../interfaces/Command';

export const Hello = {
  name: 'hello',
  description: 'Returns a greeting',
  type: ApplicationCommandType.ChatInput,
  run: (_, interaction) => {
    const username = interaction.user?.username;
    const content = `Ara ara... Hi, ${username}-kun!`;

    return interaction.reply({
      ephemeral: true,
      content,
    });
  },
} as const satisfies Command;
```